Debug App/Tests/Unit Tests/Primer/Ideal/BankComponentTests.swift

```diff
@@ -248,7 +248,7 @@ private class MockBankSelectorTokenizationModel: BankSelectorTokenizationProvidi
             .then {
                 self.fetchBanks()
             }
-            .done { banks in
+            .done { _ in
```

Comment:
I think here we should keep `banks` and pass that to `fulfill` (as `banks` is `self.mockBanks` in `fetchBanks`)